explorer/src/realtime.ts

```diff
@@ -64,21 +65,34 @@ export const bootstrapRealtime = async (server: http.Server) => {
         logger.debug(
           `websocket client successfully authenticated, new session for node ${session.chainlinkNodeId}`,
         )
-        sessions.set(info.req, session)
+        sessions.set(accessKey, session)
         callback(true, 200)
       })
       /* eslint-enable standard/no-callback-literal */
     },
   })
 
   wss.on('connection', (ws: WebSocket, request: http.IncomingMessage) => {
-    clnodeCount = clnodeCount + 1
+    // accessKey type already validated in verifyClient()
+    const accessKey = request.headers[
+      'x-explore-chainlink-accesskey'
```

Comment:
How do you feel about pulling this out as a constant? The string is used a couple of times in this file + in the tests.